x-pack/solutions/security/plugins/cloud_security_posture/public/pages/configurations/layout/findings_search_bar.tsx

```diff
@@ -44,16 +50,27 @@ export const FindingsSearchBar = ({
 
   const { dataView } = useDataViewContext();
 
+  const misconfigurationRefreshQueryKey = CDR_MISCONFIGURATIONS_INDEX_PATTERN;
+
+  const vulnerabilityRefreshQueryKey = CDR_VULNERABILITIES_INDEX_PATTERN;
+
+  const refreshQueryKey =
+    dataView.id === CDR_VULNERABILITIES_DATA_VIEW_ID_PREFIX + '-default'
```

Comment:
Doesn't this mean it would only work properly on the `default` space for vulnerabilities? Anyway, I think a more solid approach could be to have `refreshQueryKey` being passed as a required parameter in `FindingsSearchBar`.